virttest/qemu_vm.py

```diff
@@ -2893,12 +2913,14 @@ def create(self, name=None, params=None, root_dir=None,
                 logging.info("Running qemu command (reformatted):\n%s",
                              qemu_command.replace(" -", " \\\n    -"))
                 self.qemu_command = qemu_command
-                self.process = aexpect.run_tail(qemu_command,
-                                                None,
-                                                logging.info,
-                                                "[qemu output] ",
-                                                auto_close=False,
-                                                pass_fds=pass_fds)
+                monitor_exit_status = \
+                    params.get("monitor_exit_status", "off") == "on"
```

Comment:
Not sure what is current for `monitor_exit_status`, I personally think  its default should be `on`.
The rest is OK to me.